trackers/javascript-tracker/test/micro-config/iglu.json

```diff
@@ -12,6 +12,16 @@
             "uri": "http://iglucentral.com"
           }
         }
+      },
+      {
+        "name": "Github",
+        "priority": 0,
+        "vendorPrefixes": ["com.snowplowanalytics"],
+        "connection": {
+          "http": {
+            "uri": "https://raw.githubusercontent.com/snowplow/iglu-central/release/r130"
+          }
+        }
```

Comment:
Just a reminder to remove this before release, after iglu-central r130 goes live.